### DISCUSSION TITLE
sh1mmer not working

### DISCUSSION THREAD

**Discussion by vernal-ria0** [3kh0/ext-remover#531] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
![download](https://user-images.githubusercontent.com/116777371/219884662-82e379bf-8712-4a54-af43-0a223bb6e5f5.jpeg)
I have the modified shim but when I plug it in it says not a valid image. btw the computer is a hp chromebook 11 G9 EE

**Comment by funkymokey** [comment, marked as answer]
If you're using an SD card for it, i can see why

**Reply by xHydra1337** [reply]
Or maybe a usb with not enough space on it, and maybe what ever he used to flash it didnt tell him that it was too small. That and your theory are the only things I can think of, or if he just got a bad rma shim.
